src/analysis/souffle/may_will.dl

```diff
@@ -0,0 +1,45 @@
+#ifndef MAY_WILL_DL_
+#define MAY_WILL_DL_
+
+#include "taint.dl"
+
+// .decl hasInputPath(particle: Principal, path: AccessPath)
```

Comment:
Is this supposed to be here? It looks like it's some experimental code that got commented out but never deleted.

Reply:
oops. thanks.